### ISSUE TITLE
Mark the unlikely startup handling case in frbPostProcess

### ISSUE THREAD

**Issue by cubranic** [kotekan/kotekan#657]
The check `if (startup)` in the `main_thread` (line ~148) should be marked "unlikely" because it's only true once, during startup.

**Comment by andrerenard** [comment]
I don't think this is required.  The call rate on this line is only about 10Hz, and there isn't much point trying to optimize branch prediction at this level.

**Comment by cubranic** [comment]
OK. Is it useful to keep it in the pulsarPostProcess?

**Comment by andrerenard** [comment]
It's not really needed there either.

**Comment by cubranic** [comment]
Funny that #656 all started by me looking for how "likely/unlikely" in pulsarPostProces work. :-)
I'll change this issue to take them out.
